trex-operator/config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: example-cnf
+namespace: trex-operator-system
```

Comment:
Just a question, do you know if this could have an impact, now that is not in the example-cnf NS?

Reply:
I don't think so, in fact, in testpmd-operator, we were already using that kind of namespaces in the kustomization file: https://github.com/openshift-kni/example-cnf/blob/main/testpmd-operator/config/default/kustomization.yaml#L2, here I just updated the file with what was outputed by operator-sdk execution, so I believe we're fine